README.md

```diff
@@ -2,7 +2,10 @@
 
 ![Medplum](packages/docs/static/img/cover.webp)
 
-Medplum is a developer platform that enables flexible and rapid development of healthcare apps.
+
+Medplum is a comprehensive developer platform dedicated to simplifying and enhancing healthcare application development. With a strong focus on security, compliance, and interoperability, Medplum offers a range of interconnected components and tools designed to streamline the creation of healthcare applications and systems.
```

Comment:
I mostly like this change, but think it could be a bit tighter: it reads as pretty verbose to me in its current form.  Maybe something like this could work:
```suggestion
Medplum is a comprehensive developer platform that simplifies and enhances healthcare application development. With a strong focus on security, compliance, and simple integrations, Medplum provides the foundational components and tools needed to build the healthcare applications of the future.
```